examples/web-chat/src/MessageInput.tsx

```diff
@@ -1,5 +1,6 @@
 import { ChangeEvent, KeyboardEvent, useEffect, useState } from "react";
-import { useWaku } from "./WakuContext";
+import { useWaku } from "@waku/react";
+import { LightNode } from "@waku/interfaces";
```

Comment:
Would it make sense to re-export `LightNode` from `@waku/react` so that the developer only has to install `@waku/react`?
Also, it's a `type`

Reply:
It sounds like a good improvement similarly to the other one you've proposed. But I am not really sure at this point of life of the lib. On the one hand it makes sense to bundle them together, on the other the lib is just for easier integration which means for other usage one would need to have `js-waku` as a dependency. 

Also if we are speaking about types only - I think it's all right, because we already do this in JS ecosystem by using `@types/*` packages. 
nit: maybe we should rename `@waku/interfaces` to `@waku/types` to make it similar to that 🤔

Reply:
> . nit: maybe we should rename `@waku/interfaces` to `@waku/types` to make it similar to that thinking

So far, `@waku/interfaces` mainly contain interfaces useful for Waku implementer, not library consumers. It contains more type than what the consumer strictly needed. It's mostly types to enables efficient dependencies between Waku packages, ie:
- package A implements interface `foo`
- package B expects object implementing interface `foo`

package A and B can depend on `interfaces` for types purposes. No direct/circular dependency is needed between A and B.

Reply:
But it is inevitable that `@waku/interfaces` will be used by a consumer for defining something based on the interface. Of course we expect them to use their own types, following your comment, but I believe it won't always be the case.